cmd/build/helmify/main.go

```diff
@@ -101,7 +103,19 @@ func (ks *kindSet) Write() error {
 				obj = "{{- if .Values.experimentalEnableMutation }}\n" + obj + "{{- end }}\n"
 			}
 
-			if kind == "Deployment" {
+			if name == "gatekeeper-critical-pods" {
+				obj = "{{- if .Values.resourceQuota }}\n" + obj + "{{- end }}\n"
+			}
+
+			if name == "gatekeeper-controller-manager" && kind == DeploymentKind {
+				obj = strings.Replace(obj, "      priorityClassName: system-cluster-critical", "      {{- if .Values.resourceQuota }} \n      priorityClassName:  {{ .Values.controllerManager.priorityClassName }}\n      {{- end }}", 1)
```

Comment:
should this be gated on `values.controllerManager.priortyClassName`, and similar for audit?

Reply:
thank you so much, it is my bad 🤦🏻‍♂️ it should be `values.audit.priorityClassName`. I fixed it.

Reply:
I'm still seeing a test against `resourceQuota`?: `if .Values.resourceQuota`

Reply:
this time, it should be ok @maxsmythe. 🙏